test/zero-downtime/testsuite_zerodowntime_test.go

```diff
@@ -28,13 +27,14 @@ spec:
     - 
       name: hardening`
 
-const apiProbeInterval = 5 * time.Second
-const sequencesInterval = 15 * time.Second
-
 type ZeroDowntimeEnv struct {
-	quit         chan struct{}
-	NrOfUpgrades int
-	Wg           *sync.WaitGroup
+	quit              chan struct{}
+	NrOfUpgrades      int           `envconfig:"NUMBER_OF_UPGRADES" default:"2"`
+	EnvInstallVersion string        `envconfig:"INSTALL_HELM_CHART" default:"https://github.com/keptn/helm-charts-dev/raw/69eea439a26a99ecc163e296860dbb5d43e41600/packages/keptn-0.15.1-dev.tgz"`
```

Comment:
I don't know if a default here is a good approach. I would force the env var to exist

Reply:
I thought a default value would make it easier to play with it locally, but I can add it to the code as a comment and remove the default...